### PR TITLE
Fixes OpenMined #297

### LINKED ISSUE #297: Detailing the local example to match its jupyter sibling
## Where?
Where are you looking to add documentation? Which file? Which feature?
https://github.com/OpenMined/PipelineDP/blob/main/examples/restaraunt_visits/run_without_frameworks.py

## Who?
Who is your audience for this documentation issue?
Newcomers wanting to run the example locally.

## What?
Describe what you would like this documentation to accomplish.
Add details to the local example file to match its jupyter sibling (https://github.com/OpenMined/PipelineDP/blob/main/examples/restaurant_visits.ipynb) allowing newcomers to read/run either example with similar content - but strictly for a local example dataset.

### PR DESCRIPTION
**Pull request by parikhanupk**
## Description
Fixes #297
Add details to the local example file to match its jupyter sibling (https://github.com/OpenMined/PipelineDP/blob/main/examples/restaurant_visits.ipynb) allowing newcomers to read/run either example with similar content - but strictly for a local example dataset.

## Affected Dependencies
None

## How has this been tested?
python3 run_without_frameworks.py
python3 run_without_frameworks.py --output_file=output.txt

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

**Review by chinmayshah99 — CHANGES_REQUESTED** [review]
no need for `matplotlib` usage here as it becomes unintuitive in a python file. using `print` would make more sense.

**Comment by parikhanupk** [comment]
Removed matplotlib along with numpy and made changes to original file.
Allowed the example to run with or without --output_file CLI parameter (minor change).

File: examples/restaraunt_visits/run_without_frameworks.py [hunk 1]
```diff
@@ -47,8 +47,8 @@ def print_results(title, non_dp_aggregations, dp_aggregations, partition_labels)
     print(format_string.format("Day", "NonDP", "DP", "Variation %"))
     print("-"*50)
     for i in range(len(partition_labels)):
-        variation = (dp_aggregations[i]/non_dp_aggregations[i] - 1) * 100
-        print(format_string.format(partition_labels[i], non_dp_aggregations[i], dp_aggregations[i], round(variation, 3)))
+        variation = round((dp_aggregations[i]/non_dp_aggregations[i] - 1) * 100, 3) if non_dp_aggregations[i] != 0 else "NA"
```
Comment (chinmayshah99) [inline examples/restaraunt_visits/run_without_frameworks.py:50]: the check should be `non_dp_aggregations[i] - 1` != 0

**Review by chinmayshah99 — CHANGES_REQUESTED** [review]

File: examples/restaraunt_visits/run_without_frameworks.py (hunk 1 above)
Comment (parikhanupk) [reply examples/restaraunt_visits/run_without_frameworks.py:50]: Why?
The dp_aggregations[i] will be divided only by non_dp_aggregations[i]
The variation in dp with respect to nondp is (dp - nondp) / nondp which is ((dp/nondp) - 1)

Comment (chinmayshah99) [reply examples/restaraunt_visits/run_without_frameworks.py:50]: please add parenthesis to improve the readability

**Pull request closed without merging**